core/src/main/java/org/apache/iceberg/rest/auth/OAuth2Manager.java

```diff
@@ -62,6 +64,7 @@ public class OAuth2Manager extends RefreshingAuthManager {
   private long startTimeMillis;
   private OAuthTokenResponse authResponse;
   private AuthSessionCache sessionCache;
+  private ScheduledExecutorService refreshExecutor;
```

Comment:
I don't think we need any changes to this class. We would update it once `RefreshingAuthManager` has been removed right?

Reply:
Well, for me there wasn't any risk of breaking change here – but OK. Let me change this.